README.md

```diff
@@ -559,6 +560,252 @@ The model can also be downloaded and served locally.
 
    The output from the command will consist of a series of outputs from the model before and after training.
 
+### 🧪 Evaluate the newly trained model
+
+You can use the `ilab` command to evaluate the models you are training with several benchmarks. Currently, four benchmarks are supported.
+
+| Benchmark | Measures | Full Name | Description |
+| --- | --- | --- | --- |
+| MMLU | Knowledge | Massive Multitask Language Understanding | TODO |
```

Comment:
The MMLU benchmark tests a model against a standardized set of knowledge data and produces an improvement or regressions score based on this evaluation.

Reply:
I added a slightly shorter version of this as well as a reference to the MMLU paper